kuma/static/layout-tests/normal.html

```diff
@@ -0,0 +1,1460 @@
+
+
+
+<!DOCTYPE html>
+<html lang="en-US" dir="ltr" class="no-js">
+<head prefix="og: http://ogp.me/ns#">
+  <meta charset="utf-8">
+
+  <meta http-equiv="X-UA-Compatible" content="IE=Edge">
+  <script>(function(d) { d.className = d.className.replace(/\bno-js/, ''); })(document.documentElement);</script>
+  <title>Array.prototype.forEach() - JavaScript | MDN</title>
+
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="robots" content="index, follow">
```

Comment:
Let's tell robots to ignore this duplicate content.

```suggestion
  <meta name="robots" content="noindex, nofollow">
```